scene/gui/tree.cpp

```diff
@@ -2625,6 +2625,61 @@ int Tree::draw_item(const Point2i &p_pos, const Point2 &p_draw_ofs, const Size2
 	return htotal;
 }
 
+void Tree::draw_background_stripes(const Point2 &draw_ofs) {
+	const RID ci = get_canvas_item();
+
+	int stripe_count = 0;
+	TreeItem *current_item = root;
+	int total_size = draw_ofs.y - theme_cache.v_separation;
+
+	if(_get_title_button_height() > 0) {
+		stripe_count++;
+	}
+
+	// Stripes per item, respects item height.
+	while (current_item) {
+		if (const int item_height = compute_item_height(current_item); item_height > 0) {
+			const int height = item_height + theme_cache.v_separation;
+
+			// Draw every other.
+			// stripe_count - 1 to avoid an otherwise ugly gap at the top of the tree.
+			if ((stripe_count - 1) % 2 == 0) {
+				Rect2 j = Rect2(draw_ofs.x, total_size + theme_cache.v_separation, _get_content_rect().size.x, height);
+				j.position = j.position - get_scroll().floor();
```

Comment:
Did you mean to just floor the scroll?

Reply:
Yeah due to changes to fix the different vertical separation value issues I needed to floor here to avoid truncating to the wrong int value. Realise I also don't need the scroll logic for the empty stripes as if you can scroll, there is no empty space.